packages/angular/projects/clr-angular/src/data/datagrid/wrapped-column.ts

```diff
@@ -27,6 +27,15 @@ export class WrappedColumn implements DynamicWrapper, AfterViewInit, OnDestroy {
   }
 
   ngOnDestroy() {
-    this.columnView.destroy();
+    /**
+     * @TODO
+     * Check to see if Ivy is enabled and if so call destroy method.
+     * This is related to this issues #4692 #4232
+     *
+     * This check is for backward compatibility for NON-Ivy builds
+     */
+    if (this[`__ngContext__`]) {
+      this.columnView.destroy();
```

Comment:
Would it make sense to make this a separate helper function as opposed to repeating the conditional in three places?

That would also make it easier to unit test in isolation...

```
function checkIfIvyAndThen(context, fn) {
    if(context[`__ngContext__`] {
       fn();
    }
}
```

...and...

```
ngOnDestroy() {
    checkIfIvyAndThen(this, () => {
        this.columnView.destroy();
    }
}
```

Reply:
Give me a few minutes .. to find a place to put it in...